### ISSUE TITLE
Adventist Identity

### ISSUE THREAD

**Issue by KeyJoo** [RadugaCamp/diventello#72]
# Айдентика

Вопрос об использовании айдентики в изображениях (1/7 площади)
Для ознакомления, я эту тему копал 10мес. назад.

[Айдентика] (http://k5220.ru/ajdentika/)
я бы на начальном этапе, перед внедрением предложил спросить у Яши, его фидбек тоже важен!

**Comment by timofey97** [comment]
Согласен. Я не уверен что нужно вкулячать символику церкви. Я это так, для примера сделал.

**Comment by vovs03** [comment]
- https://identity.adventist.org/downloads/logo/

## RU Video YouTube :ru:
- https://www.youtube.com/watch?v=I-_6sWaH3rY Логотип Церкви Адвентистов Седьмого Дня. Сетка творения. Система идентичности.
- **FONT** Advent-sans https://identity.adventist.org/downloads/toolkit/#advent-sans

**Comment by vovs03** [comment]
### Question Как делать шрифты по таблице символов?

- https://www.urbanfonts.com/fonts/advent.font

![Image table of letters font example](https://urban-fonts.s3.amazonaws.com/samples/11143/6ee73efefff9fc9a8ac912581457fd63.jpg)

**Comment by vovs03** [comment]
### Colors
- https://alps.adventist.io/v2/?p=atoms-colors
  - 
![colors](https://user-images.githubusercontent.com/21124057/56733528-11d66100-6769-11e9-915d-3cc6e1d4e1c1.png)
  - [`Screenshot_2019-04-25_14-46-45`]

**Comment by vovs03** [comment]
# УЗОР НА КУПЮРАХ СЕРТИФИКАТАХ 

- Гильош - защитная сетка https://www.securesoft.ru/guillioche.html 
  - система для создания гильоширных элементов (защитные сетки, розетты, бордюры) любой степени сложности.
- Гильоширование
- Гильоширная сетка

https://www.youtube.com/results?search_query=guillioche+Inkskape

## Soft

- https://www.securesoft.ru/glissando.html (Программа Глиссандо)

![Глиссандо образец](https://www.securesoft.ru/images/glissando/glissando_logo.gif)

## Примеры Видео

- https://www.youtube.com/watch?v=9RAVo15lxJ4 [Делаем защитную сетку на документы]
- https://www.youtube.com/watch?v=sizMHyRXpXw [Скачай бесплатно прямо сейчас. Защитная сетка. Гильоши. Часть 3. #Корел. #Corel]

### guillioche

- https://www.youtube.com/results?search_query=guillioche+Inkskape
- https://m.youtube.com/watch?v=BahYdJq4L7U

## Тангирная сетка (несколько цветов)

- https://www.youtube.com/watch?v=Nxbww3SywfM [Тангирная сетка с силуэтом.]

![Screenshot_2019-04-25_16-41-23](https://user-images.githubusercontent.com/21124057/56740142-04c16e00-6779-11e9-85ce-aa71a414b829.png)
